ui/test/nuxeo-layout.test.js

```diff
@@ -47,4 +48,36 @@ suite.skip('<nuxeo-layout>', () => {
     expect(layout.element.localName).to.equal('dummy-layout');
     expect(layout.element.text).to.equal('dummy');
   });
+
+  test('layout valid field invalid', async () => {
```

Comment:
nit: this test tests exactly the same as the third one, right? (required with a value equal to `invalid` vs `foo`)

Reply:
oops tx